packages/lib/OgImages.tsx

```diff
@@ -58,12 +58,12 @@ const makeAbsoluteUrl = (url: string) => (/^https?:\/\//.test(url) ? url : `${CA
  */
 export const constructMeetingImage = ({ title, users = [], profile }: MeetingImageProps): string => {
   const url = [
```

Comment:
Can we not encode the entire URL parameter instead of each individual component?

Reply:
addressed